src/services/rfqm_service.ts

```diff
@@ -118,8 +118,9 @@ export class RfqmService {
         const marketOperation = isSelling ? MarketOperation.Sell : MarketOperation.Buy;
         const assetFillAmount = isSelling ? sellAmount! : buyAmount!;
 
-        // Prepare gas estimate
+        // Prepare gas estimate and fee
         const gas: BigNumber = await this._protocolFeeUtils.getGasPriceEstimationOrThrowAsync();
+        const feeAmount = gas.times(RFQM_TX_GAS_ESTIMATE);
```

Comment:
I think this will be in gwei, should we convert this to wei?

Reply:
name edit => gasPrice

Reply:
asset-swapper doc says:

```
    private async _getGasPriceFromGasStationOrThrowAsync(): Promise&lt;BigNumber&gt; {        try {            const res = await fetch(this._ethGasStationUrl);            const gasInfo = await res.json();            // Eth Gas Station result is gwei * 10            // tslint:disable-next-line:custom-no-magic-numbers            const BASE_TEN = 10;            const gasPriceGwei = new BigNumber(gasInfo.fast / BASE_TEN);            // tslint:disable-next-line:custom-no-magic-numbers            const unit = new BigNumber(BASE_TEN).pow(9);            const gasPriceWei = unit.times(gasPriceGwei);            // Reset the error count to 0 once we have a successful response            this._errorCount = 0;            return gasPriceWei;        } catch (e) {            this._errorCount++;            // If we've reached our max error count then throw            if (this._errorCount &gt; MAX_ERROR_COUNT || this._gasPriceEstimation.isZero()) {                this._errorCount = 0;                throw new Error(SwapQuoterError.NoGasPriceProvidedOrEstimated);            }            return this._gasPriceEstimation;        }    }
```

This will return wei as is, which I think is good.

Reply:
you right